lib/transport/selenium-webdriver/options.js

```diff
@@ -111,6 +111,23 @@ module.exports = class SeleniumCapabilities {
     return options;
   }
 
+  createIeOptions() {
+    const {Options: IeOptions} = require('selenium-webdriver/ie');
+
```

Comment:
`desiredCapabilites` can already be an instance of EdgeOptions class we need a check for that.
```suggestion
 if (this.alreadyDefinedAs(IeOptions)) {
      return this.desiredCapabilities;
  }
```

Reply:
We can also add a test covering this.

Reply:
Addressed in latest commit